01_explore_data/nbconverted/explore_cancer_gene_sets.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Explore cancer gene sets
+# 
+# In `00_download_data/2_download_cancer_gene_data.ipynb`, we downloaded and integrated several different sets of cancer-associated genes. In this script, we want to explore the similarities and differences between these gene sets in more detail.
+# 
+# We see that the results of our classification experiments are somewhat dependent on which gene set or sets we decide to use, so understanding the similarities and differences between the gene sets will help us to put these in context.
```

Comment:
Maybe helpful to link to the different publication/DB sources where you're downloading the data from.

Also, I assume the motivation for looking at these different input datasets is to see if your results/prediction trends generalize. Is that correct

Reply:
Maybe helpful to add a short description of the different gene sets - i.e. number of total genes, what type of patients and how they determined oncogene vs non-oncogene

Reply:
I do have most of this information in the script at `00_download_data/2_download_cancer_gene_data.ipynb` where we actually download the data - I'll make it clear here that there's more information about the gene sets in that script.

> Also, I assume the motivation for looking at these different input datasets is to see if your results/prediction trends generalize. Is that correct

Yep! Both reviewers on our paper wanted to see if our conclusions generalized outside of the Vogelstein dataset, particularly to a dataset with more DNA damage repair genes (which COSMIC CGC has a lot of). Turns out it does change our conclusions a bit, so it was a helpful review comment!